sanitize/src/lib.rs

```diff
@@ -36,6 +43,7 @@ pub trait Sanitize {
     }
 }
 
+#[cfg(feature = "alloc")]
 impl<T: Sanitize> Sanitize for Vec<T> {
```

Comment:
Sorry that I'm only just now thinking of this, but we could change this to a no-std compliant variant by making it: `impl<T: Sanitize> Sanitize for [T]`.

Unsized types can be a bit annoying to work with, but this makes everything Just Work. What do you think?